i2c/pcf8523_i2c/pcf8523_i2c.c

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/i2c.h"
+
+/* Example code to talk to a PCF8520 Real Time Clock module 
```

Comment:
I dunno if it's deliberate, but why is "PCF8523" mentioned in some places, but "PCF8520" mentioned in other places?